### DISCUSSION TITLE
https://dimensium.github.io/docs/dimensium-docs/writting-docs/

### DISCUSSION THREAD

**Discussion by giscus** [dimensium/docs#1] [Giscus discussions]
# https://dimensium.github.io/docs/dimensium-docs/writting-docs/

The official documentation repository of Dimensium, using mkdocs with material theme for mkdocs.

https://dimensium.github.io/docs/dimensium-docs/writting-docs/

**Comment by Promesis** [comment]
# Q&A

**Any questions, suggests, and answers are welcomed. Be free to talk about anything.**

Put your question here!
